app/assets/stylesheets/mobile.scss

```diff
@@ -1,6 +1,6 @@
 @import "_defaults";
 
-@media (max-width: 500px) {
+@media (max-width: 31.25 * $root-font-size) {
```

Comment:
I think you told me about why it makes sense to do it this way, but now I can't remember why. please explain again?

Reply:
There's a good article here https://www.sitepoint.com/understanding-and-using-rem-units-in-css/ that references another article, but scroll down to the *Using rems with media queries breakpoints* section, and it explains the reasoning behind this. This really talks about the "optimal line length" and it's something that used in blogs / articles / etc. 

i guess the same thing applies here, but i think there's this concern of "are we just converting everything to multiples of 16 for no particular reason?" that is maybe a little valid.

There's standard breakpoints people use, something like:
mobile: < 768
tablet: >= 768 < 992
small desktop: >=992 < 1024
desktop: >=1200 < 1600
xl desktop >= 1600

these really have nothing to do with what your root font size is. these are based on like the actual physical device screen sizes, independent of whether you use 16 or 18 or 14.

so breakpoints and things like that should not necessarily be tied to your root font size in all cases.